packagebuild/daisy_startupscript_deb.sh

```diff
@@ -38,10 +40,19 @@ sed -i 's/^.*debian buster-backports main.*$//g' /etc/apt/sources.list
 
 try_command apt-get -y update
 try_command apt-get install -y --no-install-{suggests,recommends} git-core \
-  debhelper devscripts build-essential equivs libdistro-info-perl
+  debhelper devscripts build-essential equivs libdistro-info-perl protobuf-compiler
+install_protoc
```

Comment:
Is `unzip`/`make` already installed? You might want to add that if not already installed, I had to install it for googet packaging

Reply:
`unzip` isn't, but `make` is as it's a part of `build-essential`. Added `unzip` to the list of installs.